### ISSUE TITLE
Accessibility by screen readers feature.

### ISSUE THREAD

**Issue by mansour-ahmed** [swimlane/ngx-charts#990]
**I'm submitting a ...**  (check one with "x")

- [ ] bug report - search github for a similar issue or PR before submitting
- [x] feature request
- [ ] support request - use StackOverflow (add the `ngx-charts` tag) or the gitter chat for support questions

**Current behavior**
The charts are currently not accessibly using a screen reader. The screen reader reads the legends and the axis values, however it does not pick up the reading values highlighted by keyboard for instance in a bar chart a highlighted bar. The screen reader just highlight that its a group but does not read out the value itself.

See screenshot for more information.
<img width="1679" alt="screenshot 2018-09-28 at 14 38 33" src="https://user-images.githubusercontent.com/12623964/46209189-688fcf80-c32d-11e8-8c55-87609ceafbb4.png">

**Expected behavior**
It would allow the screen reader to pick the data displayed by the graph.

**Reproduction of the problem**
1. navigate to https://swimlane.github.io/ngx-charts/#/ngx-charts/bar-vertical
2. Switch on the screen reader (for mac os you can use the shortcut cmd+f5)
3. try to read a value using the screen reader, for instance GDP of Germany. 

**What is the motivation / use case for changing the behavior?**
To allow the charts to be accessible by people with limitations/disabilities, WCAG compliance & also allow crawlers to read the graph's data that belongs SEO.
**Please tell us about your environment:**
Chrome 69, MacOS (Mojave)

* **ngx-charts version:** 9.0.0

* **Angular version:** Not sure

* **Browser:** [ all ]

**Comment by patelnaman6** [comment]
Is there any update on the issue?

**Comment by BillLaNTT** [comment]
Hey, I am working on an a11y WCAG compliance request for the generated SVG to have the attributes:
role="img"
aria-label="chart"
in lieu of the roadmap being implemented, is the best way to do this in the short term to create a custom a chart with those attributes and implement that?

**Comment by JGuinto-QuantumIT** [comment]
please we also need this feature.

**Comment by sjdrew** [comment]
Us too.

**Comment by SiddanthTantry** [comment]
We need it too. Any update on this?
